about.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="index.css" rel="stylesheet" type="text/css">
+    <style type="text/css"></style>
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="grid-wrapper">
+        <header class="masthead">
+            <nav class="toolbar">
+                <ul>
+                    <li style="padding-top: 0.5rem;"><a href="https://www.facebook.com/events/1455321544773645/"><img
+                                src="statics/images/facebook.png" alt="Facebook"></a></li>
+                    <li style="padding-top: 0.5rem;"><a href="https://twitter.com/cckorea"><img
+                                src="statics/images/twitter.png" alt="Twitter"></a>
+                    </li>
+                    <li><a href="https://summit.creativecommons.org/" target="_blank">Logout</a></li>
+                    <li><a href="https://summit.creativecommons.org/" target="_blank">Mypage</a></li>
+                    <li><a href="https://summit.creativecommons.org/" target="_blank">Bengali</a></li>
+                </ul>
+
+            </nav>
+            <div class="masthead_inner">
+                <img class="mobile_menu" src="statics/images/mobile_menu.png" alt="Menu" />
+
+                <div class="mylogo">
+                    <h1><a href="/technothirst">TechnoThirst</a></h1>
+                </div>
+                <div>
+                    <ul class="nav_link">
+                        <li><a href="./about.html">About</a></li>
+                        <li><a href="#">Write</a></li>
+                        <li><a href="#">Sign in</a></li>
+                        <li><a href="#"> Contact</a></li>
+                        <li class="live"><a href="#presentation">Presentation</a></li>
+                    </ul>
+                </div>
```

Comment:
[Optional] Kindly include a link to the home page from the about page for easy navigation to the home page from the about page.

Your about page:
![image](https://user-images.githubusercontent.com/20554583/211142637-ca7079d6-e707-4e29-8d32-eebf04f9e4cb.png)